home/utils.py

```diff
@@ -219,6 +219,10 @@ def send_updates_webhook(request, *, include_professors=True, include_reviews=Tr
         url=request.build_absolute_uri(reverse("admin")))
 
     webhook.add_embed(embed)
+
+    if num_reviews + num_professors % WEBHOOK_FREQUENCY != 0:
+        return
```

Comment:
this isn't a perfect solution since unverified professors and reviews can both increment at the same time if someone reviews a new professor. But I don't really want to deal with adding a new DB table to track this, so let's go with it for now.

Reply:
The fact that reviews and professors can increment together is irrelevant because the point of this change is to be notified once there are a total of X new notifications

Reply:
I understand, but if num_reviews + num_professors was at 19, and someone reviews a new professor, it will then be at 21 and we will not get a notification for 20.

Reply:
Ah good point. But I agree with you that fixing it isn't worth the effort. In that case, guess we'll have to wait 9 more reviews to be notified.